state-chain/pallets/cf-swapping/src/tests.rs

```diff
@@ -1791,21 +1799,34 @@ mod on_chain_swapping {
 				);
 
 				// Now we adjust execution price so that the next chunk gets refunded:
-				SwapRate::set(DEFAULT_SWAP_RATE as f64 / 2.0);
+				SwapRate::set(NEW_SWAP_RATE);
 			})
 			.then_process_blocks_until_block(CHUNK_2_BLOCK)
 			.then_execute_with(|_| {
+				const REFUND_FEE: AssetAmount = MIN_NETWORK_FEE * NEW_SWAP_RATE as u128;
 				// Only one chunk is expected to be swapped:
 				const EXPECTED_OUTPUT_AMOUNT: AssetAmount =
-					CHUNK_AMOUNT * DEFAULT_SWAP_RATE * DEFAULT_SWAP_RATE;
+					CHUNK_AMOUNT * DEFAULT_SWAP_RATE * DEFAULT_SWAP_RATE -
```

Comment:
Nit: I would group like this to make it clear that network fee is deducted between leg 1 and leg 2:

`(CHUNK_AMOUNT * DEFAULT_SWAP_RATE - MIN_NETWORK_FEE) * DEFAULT_SWAP_RATE`.

Also, I think it is a little subtle why we charge min network fee rather than the usual 10bps. I'm assuimg it is because the chunk amounts are too small and/or that network fee is currently set to 0? I think it is woth adding a comment clarifying this.